SpectatorView/Samples/SharedHolograms/Assets/Addons/HolographicCameraRig/SV_UNET/Scripts/PlayerController.cs

```diff
@@ -389,7 +389,7 @@ private void InitializeSpectatorView()
                 !SpectatorView.SpectatorViewManager.Instance.IsCurrentlyActive &&
                 IsSV())
             {
-                SpectatorView.SpectatorViewManager.Instance.EnableHolographicCamera(UNetAnchorManager.Instance.gameObject.transform);
+                SpectatorView.SpectatorViewManager.Instance.EnableHolographicCamera(this.gameObject.transform);

```

Comment:
HolographicCamera is deliberately not a child of the player because of how Unity treats children of NetworkBehaviours when the parent object goes out of scope.  By making it it's own child object, when the spectator view HoloLens goes to sleep, spectator view can still be used from the last known position.  This is very helpful for stage demos where you do not want the holograms going out of scope!

See: https://github.com/Microsoft/MixedRealityCompanionKit/blob/master/SpectatorView/Samples/SharedHolograms/Assets/Addons/HolographicCameraRig/Scripts/SpectatorViewManager.cs#L264